forge-gui-mobile/src/forge/adventure/scene/InnScene.java

```diff
@@ -39,13 +59,17 @@ private InnScene() {
         leaveIcon = ui.findActor("leaveIcon");
         healIcon = ui.findActor("healIcon");
         sellIcon = ui.findActor("sellIcon");
+
+        event = ui.findActor("event");
+        eventDescription = ui.findActor("eventDescription");
+
+        ui.onButtonPress("event", InnScene.this::startEvent);
     }
 
 
 
     public void done() {
-        GameHUD.getInstance().getTouchpad().setVisible(false);
-        Forge.switchToLast();
+        Forge.switchScene(lastGameScene==null?GameScene.instance():lastGameScene);
```

Comment:
why remove this: GameHUD.getInstance().getTouchpad().setVisible(false); does the innscene don't display the touchpad on android anymore when this is called?

Reply:
Unintentional removal during replacement of the next line. Will put this back.